tests/fixtures/yaml_dicts/_layouts/index.liquid

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>My blog - {{ title }}</title>
+</head>
+<body>
+{{ content }}</body>
+</html>
```

Comment:
Agreed that the testing is less than ideal right now.  We'd like to move more towards unit testing rather than end-to-end testing but we need to decouple some of the pieces so we can unit test them.